packages/mongo-nav/src/data.ts

```diff
@@ -0,0 +1,51 @@
+import { DataInterface } from './types';
+
+export default {
+  account: {
+    email: 'harry.wolff@mongodb.com',
```

Comment:
Gave it a stab, if you want the data to be more specific, include something in particular lmk. Otherwise just named orgs/projects Demo Org/Demo Project